java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -804,7 +809,7 @@ void testFromScalar() {
   void testFromScalarNull() {
     final int rowCount = 4;
     for (DType type : DType.values()) {
-      if (type == DType.EMPTY) {
+      if (type == DType.EMPTY || type == DType.LIST) {
```

Comment:
We need a follow on issue to support scalar lists.

Reply:
Yes.